_posts/2025-06-30-probabl-skolar.md

```diff
@@ -0,0 +1,93 @@
+---
+#### Blog Post Template ####
+
+#### Post Information ####
+title: "Skolar: an open-source initiative to democratize open data science"
+date: June 30, 2025
+
+#### Post Category and Tags ####
+# Format in titlecase without dashes (Ex. "Open Source" instead of "open-source")
+categories:
+  - Updates
+tags:
+  - Sponsor
+  - Open Source
+  - Machine Learning
+
+#### Featured Image ####
+featured-image: BSD_watermark.svg
+
+#### Author Info ####
+# Can accomodate multiple authors
+# Add SQUARE Author Image to /assets/images/author_images/ folder
+postauthors:
+  - name: Skolar
+    website: https://skolar.probabl.ai/
+    image: "skolar-logo.png"
+  - name: Pénélope Gittos
+    website: https://www.linkedin.com/in/gittospenelope-data-analyst-growth-bilingual/
+    image: "penelope_gittos.jpeg"
+---
+<div>
+  <img src="/assets/images/posts_images/{{ page.featured-image }}" alt="">
+  {% include postauthor.html %}
+</div>
+
+<span style="color:red">*This blog post has been submitted by Probabl, a sponsor of scikit-learn.* </span>
+The scikit-learn project always puts efforts on education to build and nurture a
```

Comment:
A few suggestions, even if they say something slightly different to my interpretation of the original
```suggestion
The scikit-learn project values educational efforts that build and nurture a
```

```suggestion
The scikit-learn project has always put effort into education in order to build and nurture a
```

```suggestion
The scikit-learn project has always put a lot of effort into education in order to build and nurture a
```

(I don't know if posting three suggestions will work or not, maybe don't use the GitHub UI to apply it?)